digitaltwins/samples/model_repository/delete_model.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const ModelRepositoryCredentials = require('azure-iot-digitaltwins-model-repository').ModelRepositoryCredentials;
+const ModelRepositoryServiceClient = require('azure-iot-digitaltwins-model-repository').ModelRepositoryServiceClient;
+
+function replacer(key, value) {
+  if (key == '_response') {
+    return undefined;
+  } else {
+    return value;
+  }
+}
+
+async function main() {
```

Comment:
same here. What is the purpose of this sample? What are the calls doing? Can I change modelId to 'foo' or does it need to be in the format given? Not much is clear.